django/conf/global_settings.py

```diff
@@ -543,6 +543,8 @@ def gettext_noop(s):
 CSRF_COOKIE_PATH = '/'
 CSRF_COOKIE_SECURE = False
 CSRF_COOKIE_HTTPONLY = False
+CSRF_USE_SESSIONS = False
+CSRF_SESSION_KEY = 'csrftoken'
```

Comment:
Is there any strong argument to make this configurable and not just have something like https://github.com/django/django/blob/master/django/contrib/auth/__init__.py#L14 ?